torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -734,6 +734,15 @@ RegisterOperators reg(
            return 0;
          },
          aliasAnalysisConservative()),
+     Operator(
+         "aten::requires_grad_(Tensor(a!) self, bool _requires_grad=True) -> Tensor(a!)",
```

Comment:
@pbelevich hey any reason we have this schema with `_requires_grad` instead of `requires_grad`? This is creating discrepancy between the jit api and the python api as it take `requires_grad`..